### DISCUSSION TITLE
Need to implement tailwindcss into Microsoft Word Add in React

### DISCUSSION THREAD

**Discussion by lhlRahman** [tailwindlabs/tailwindcss#13090] [Help]
Hey,  this is my first ever opened issue so I am sorry if I missed something.

I have been trying to add tailwindCss into Microsoft word add-ins in react, I have followed the following  : https://stackoverflow.com/questions/73561810/how-to-setup-tailwind-css-with-a-react-microsoft-office-add-in-created-with-yeom
however it still does not work, here is a link to my code: https://github.com/lhlRahman/Word-Add-In

thank you all

<img width="995" alt="Screenshot 2024-03-05 at 1 20 51 PM" src="https://github.com/tailwindlabs/tailwindcss/assets/136003890/1cfd12bf-b97b-4498-837f-f1dd05ec30bb">

**Comment by wongjn** [comment]
I ran `build`, `build:dev`, `dev-server` and `watch` npm scripts with no errors. Consider checking for any uncommitted or not-yet-pushed changes that could be causing the error. You could also consider checking you have restarted any development process since making any Webpack configuration changes.
